dashboard-widgets.el

```diff
@@ -368,10 +368,7 @@ If MESSAGEBUF is not nil then MSG is also written in message buffer."
              (dashboard-get-banner-path 1)))
           ((integerp dashboard-startup-banner)
            (dashboard-get-banner-path dashboard-startup-banner))
-          ((and dashboard-startup-banner
-                (image-type-available-p (intern (file-name-extension
-                                                 dashboard-startup-banner)))
-                (display-graphic-p))
+          ((stringp dashboard-startup-banner)
```

Comment:
This might sounds stupid, but I changed to check if the value is string and ignore to check if it is image and `display-graphic-p`. So just let the user decide what file to load? Just mentioned prefer setting it this way.

```el
(setq dashboard-startup-banner (if (display-graphic-p) "path/to/image.png" "path/to/text.txt")
```

Reply:
I like your suggestion on how to setup `dashboard-startup-banner`, I would like to see the `README.md` updated with that. 

But this line change would break backwards compatibility. Until now, people set `dashboard-startup-banner` like this:
```elisp
(dashboard-startup-banner "/tmp/emacs-dashboard/banners/logo.png")
```

If you start emacs in terminal with the variable like set like that, it would fail to start. I think you should handle what happens if the value is a `png` file but emacs is started with now window system.

![image](https://user-images.githubusercontent.com/8639179/61587426-9a0eab80-ab8a-11e9-9c2c-15cf09ac5161.png)